ROADMAP.md

```diff
@@ -38,4 +36,5 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
 
 ## Rook 1.0
 
-* Declare numerous types as v1
\ No newline at end of file
+* Declare numerous types as v1
+* Automated upgrades
```

Comment:
So this is saying that the first version that you can automatically upgrade to (deploy the new operator and it takes care of it for you) will be the 1.0 version?

Reply:
right, since we would be declaring beta in 0.9, the automatic upgrades wouldn't come until 1.0